### ISSUE TITLE
src/locales/*/address/city.ts is inplausible

### ISSUE THREAD

**Issue by ST-DDT** [faker-js/faker#1471]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/en_AU_ocker/address/city.ts#L1

It should most likely point to `city_name` or be deleted.

**Note:** After #1469 is merged these files will then be located under `src/locales/*/location/city.ts`.

**Comment by ST-DDT** [comment]
On hold till #1469 is fixed.

**Comment by ST-DDT** [comment]
Also

https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/en_NG/address/city.ts#L1

**Comment by ST-DDT** [comment]
Also

https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/en_ZA/address/city.ts#L1

**Comment by ST-DDT** [comment]
Also

https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/es/address/city.ts#L1

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/es_MX/address/city.ts#L1

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/fr_BE/address/city.ts#L1-L4

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/nl_BE/address/city.ts#L1-L4

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/sv/address/city.ts#L1

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/zh_CN/address/city.ts#L1

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/zh_TW/address/city.ts#L1

**Comment by ST-DDT** [comment]
I might have missed some files, so the other `city.ts` files should be checked again.

**Comment by matthewmayer** [comment]
Here are all the different patterns in use

```
{{company.adjective}}{{company.noun}}{{location.city_suffix}} : dv
{{location.city_name}} : ar,az,cz,de_AT,de_CH,dv,en,en_GH,en_IN,fa,fr,fr_CH,he,hr,id_ID,lv,mk,ne,pl,pt_PT,ro,ru,sk,tr,uk,vi
{{location.city_name}}{{location.city_suffix}} : ko,nb_NO
{{location.city_prefix}} : en_AU_ocker,en_NG,en_ZA,es,es_MX,fr_BE,nl,nl_BE
{{location.city_prefix}} {{company.noun}}{{location.city_suffix}} : dv
{{location.city_prefix}} {{person.first_name}} : de,dv,en,ge,it,ur
{{location.city_prefix}} {{person.first_name}} {{location.city_suffix}} : it
{{location.city_prefix}} {{person.first_name}}{{location.city_suffix}} : de,en,ge,ur
{{location.city_prefix}} {{person.firstName}} : af_ZA,el,en_AU,en_CA,en_IE,en_US,fr_CA,ne,pt_BR,ro,tr,zu_ZA
{{location.city_prefix}} {{person.firstName}}{{location.city_suffix}} : af_ZA,el,en_AU,en_CA,en_IE,en_US,fr_CA,ne,pt_BR,ro,tr,zu_ZA
{{location.city_prefix}} {{person.last_name}} : hy
{{location.city_prefix}} {{person.lastName}} : en_GB
{{location.city_prefix}} {{person.lastName}}{{location.city_suffix}} : en_GB
{{location.city_prefix}} {{person.male_first_name}} : uk
{{location.city_prefix}}{{location.city_suffix}} : fr_BE,nl,nl_BE,sv,zh_CN,zh_TW
{{location.city_prefix}}{{person.first_name}}{{location.city_suffix}} : ja
{{location.city_prefix}}{{person.last_name}}{{location.city_suffix}} : ja
{{person.first_name}} {{location.city_suffix}} : it
{{person.first_name}}{{location.city_suffix}} : de,dv,en,ge,ja,ur
{{person.firstName}}{{location.city_suffix}} : af_ZA,el,en_AU,en_CA,en_IE,en_US,fr_CA,ne,pt_BR,ro,tr,zu_ZA
{{person.last_name}} {{location.city_suffix}} : it
{{person.last_name}}{{location.city_suffix}} : de,dv,en,ge,hy,ja,ur
{{person.lastName}}{{location.city_infix}}{{person.lastName}} : en_GB
{{person.lastName}}{{location.city_suffix}} : af_ZA,el,en_AU,en_CA,en_GB,en_IE,en_US,fr_CA,ne,pt_BR,ro,tr,zu_ZA
```

These are the only ones which don't include city_name or last or first names as a basis.
```
{{company.adjective}}{{company.noun}}{{location.city_suffix}} : dv
{{location.city_prefix}} {{company.noun}}{{location.city_suffix}} : dv
{{location.city_prefix}}{{location.city_suffix}} : fr_BE,nl,nl_BE,sv,zh_CN,zh_TW
{{location.city_prefix}} : en_AU_ocker,en_NG,en_ZA,es,es_MX,fr_BE,nl,nl_BE
```

These probably need manual checking, for example for en_NG the city_prefix file should just be renamed to city_name, but for example for `sv` i think its correct, its generating a fake city name by combining various prefixes and suffixes.

**Comment by xDivisionByZerox** [comment]
Resolved by #2094.
